src/DataFactory/DataFactoryV2/IntegrationRuntimes/SetAzureDataFactoryIntegrationRuntimeCommand.cs

```diff
@@ -279,6 +279,51 @@ public class SetAzureDataFactoryIntegrationRuntimeCommand : IntegrationRuntimeCm
             IgnoreCase = true)]
         public string Edition { get; set; }
 
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByIntegrationRuntimeName,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyIntegrationRuntimeName)]
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByResourceId,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyIntegrationRuntimeName)]
+        [Parameter(
+            ParameterSetName = ParameterSetNames.ByIntegrationRuntimeObject,
+            Mandatory = false,
+            HelpMessage = Constants.HelpIntegrationRuntimeDataProxyIntegrationRuntimeName)]
+        [ValidateNotNullOrEmpty]
+        public string DataProxyIntegrationRuntimeName { get; set; }
```

Comment:
How does the customer get this value? Does it make sense to have a static set of values or a dynamic completer?

Reply:
it's the name of existed selfhosted IR. Customer should know it